### PR TITLE
Kin reset

### PR DESCRIPTION
**Pull request by roystgnr**
Rebased against post-#102 master.  There were a couple rebase conflicts, so this deserves more eyeballs before we merge, even if the testing goes okay.

### CODE REVIEW

**Comment by roystgnr** [comment]
This should supercede #84

**Comment by roystgnr** [comment]
Needs a couple more deprecated shim methods before merging.

File: test/Makefile.am [hunk 1]
```diff
@@ -176,6 +178,8 @@ TESTS += units_unit
 TESTS += kinetics_reactive_scheme_unit.sh
 TESTS += sigma_bin_converter_unit
 TESTS += sigma_bin_converter_vec_unit
+TESTS += sigma_bin_converter_unit.sh
```
Comment (SylvainPlessis) [inline test/Makefile.am]: This line should not be.  There is no such test nor file.  I have no idea of the history of it...

**Comment by roystgnr** [comment]
This passes make check (by itself and with GRINS and FIN-S) for me now.  Give my additions a look and then we can merge?

**Comment by SylvainPlessis** [comment]
All good to me

**Comment by pbauman** [comment]
:+1:  Thanks!

**Pull request merged**